lib/infrastructure/contact_form/contact_form_api.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:collaction_app/domain/i_settings_repository.dart';
+import 'package:http/http.dart' as http;
+import 'package:injectable/injectable.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+import '../../domain/contact_form/contact_form_contents.dart';
+import '../../domain/contact_form/i_contact_form_api.dart';
+
+@LazySingleton(as: IContactFormApi)
+class ContactFormApi extends IContactFormApi {
+  final http.Client client;
+  final ISettingsRepository settingsRepository;
+  ContactFormApi(this.client, this.settingsRepository);
+
+  @override
+  Future<bool> sendContactFormContents(ContactFormContents contents) async {
+    assert(contents.email != null);
+    assert(contents.message != null);
+    final platform =
```

Comment:
Remove this and just use `Platform.operatingSystem` defined as:
`A string (linux, macos, windows, android, ios, or fuchsia) representing the operating system.`